### PR TITLE
Added keyboard layout in X page from old wiki

### PR DESCRIPTION
**Pull request by ectlunya**

### CODE REVIEW

File: xorg/keyboard-layout.txt [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+Use setxkbmap to set keyboard layout preferences.
+
+* Build and install setxkbmap.
+
++------------------------------------------------------------------------------+
+|                                                                              |
+|   $ kiss b setxkbmap                                                         |
+|   $ kiss i setxkbmap                                                         |
+|                                                                              |
++------------------------------------------------------------------------------+
```
Comment (dylanaraps) [inline xorg/keyboard-layout.txt:17]: This should match the style of the other pages. See https://k1ss.org/wiki/help/page-template

**Comment by ectlunya** [comment]
Are there any new fonts to add to the list?

**Comment by dylanaraps** [comment]
This page is a little bare to me. Is there anything we can do to expand it? How does one figure out which

**Comment by dylanaraps** [comment]
:+1:

**Pull request merged**